docs/configuration.md

```diff
@@ -1,8 +1,7 @@
 # Configuration
 
-The configuration for `jsii` is recorded in the `package.json` file, which is
-the standard package manifest for NPM packages. This document describes the
-constraints and extensions `jsii` adds to the [package.json schema].
+The configuration for `jsii` is recorded in the `package.json` file, which is the standard package manifest for NPM
+packages. This document describes the constraints and extensions `jsii` adds to the [package.json schema].
```

Comment:
(for the future) prefer wholesale adjustments and refactors to be separate PRs, hence separate commits 😉

Reply:
Yeah this is auto-format-on-save... -_-